docs/prover/cli/options.md

```diff
@@ -967,34 +971,113 @@ error on encountering contract method calls in quantified expression bodies.
 
 
 (control-flow-splitting-options)=
-Advanced options that control control flow splitting
-----------------------------------------------------
+Control flow splitting options
+------------------------------
 
 See [here](control-flow-splitting) for an explanation of control flow splitting.
 
+(-depth)=
+### `--prover_args '-depth <number>'`
+
+**What does it do?**
+
+Sets the maximum splitting depth.
+
+**When to use it?** 
+
+When the deepest {term}`split`s are too heavy to solve, but not too high in
+number, increasing this will lead to smaller, but more {term}`split leaves`, which run
+at the full SMT timeout (as set by {ref}`--smt_timeout`). Conversely, if run
+time is too high because there are too many splits, decreasing this number means
+that more time is spent on fewer, but bigger split leaves.
```

Comment:
1. do we have a tip on when to increase or decrease? 
2. should we state the current default (10)?

Reply:
re 1: well .. this is supposed to be it ... but it's rather abstract, I admit .. I'll give it another shot
re 2: sure (increased maintenance effort / failure potential is a con, but I guess I'll add it and see how it goes ..)

Reply:
hm, actually, I'm still not sure what to write, as changing this should really be a reaction to where CVT is spending its time ... which currently can only be observed in statsdata.json ...
however, there are plans to condense this info to a live stat -- so I propose to update this when introducing these stats.